lib/vaultex/redirectable_requests.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Vaultex.RedirectableRequests do
+  # Is there a better way to get the default HTTPoison value? When this library is consumed by a Client
+  # the config files in Vaultex appear to be ignored.
```

Comment:
Yeah, we've come across this as well... The work around we used for now was to set the config at the consuming level, but that's nasty!